src/service/restriction.service.ts

```diff
@@ -0,0 +1,82 @@
+/*
+ *
+ * Copyright 2018-present NEM
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import {AccountRestrictionFlags, MosaicRestrictionType} from 'nem2-sdk';
+
+export class RestrictionService {
+    constructor() {
+
+    }
+
+    public static getMosaicRestrictionType(type: string): MosaicRestrictionType {
```

Comment:
The switch statements looks unecessary, how about:

> return MosaicRestrictionType[type];

Reply:
We could delete this service and follow a similar approach to the ``NetworkResolver`` by using the nem2-sdk enum directly: 

https://github.com/nemtech/nem2-cli/pull/175/files#diff-2ff3a0e454bb1fc196e9f30ae70e6dd0R22